r/R/util.R

```diff
@@ -110,3 +110,15 @@ handle_embedded_nul_error <- function(e) {
   }
   stop(e)
 }
+
+handle_parquet_io_error <- function(e, format) {
```

Comment:
You aren't using `format` in here anymore--shouldn't you? Assuming you delete the format handling I suggested above, in here you will know that `format` is default if `is.character(format) && length(format) > 1` (i.e. `c("parquet", "arrow", "ipc", "feather", "csv", "tsv", "text")`)

Reply:
Good point!